src/rp2_common/pico_async_context/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "pico_async_context",
+    srcs = [
+        "async_context_base.c",
+        "async_context_freertos.c",
+        "async_context_poll.c",
+        "async_context_threadsafe_background.c",
+    ],
+    hdrs = [
+        "include/pico/async_context.h",
+        "include/pico/async_context_base.h",
+        "include/pico/async_context_freertos.h",
+        "include/pico/async_context_poll.h",
+        "include/pico/async_context_threadsafe_background.h",
+    ],
+    includes = ["include"],
+    # Missing deps for:
```

Comment:
Will all of these come from FreeRTOS? Another reason to upstream our FreeRTOS build file (https://pwbug.dev/326625641)!

Reply:
Yeah, these are all provided by FreeRTOS. Interestingly I'm not seeing CMake configuration options for turning this on/off despite it seeming to be an optional feature. I'll mark this on my TODO list.